packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -306,6 +308,24 @@ class Slider extends React.Component {
     });
   };
 
+  handleMouseEnter = event => {
+    // If the slider was being interacted with but the mouse went off the window
+    // and then re-entered while unclicked then end the interaction.
+    if (event.buttons === 0) {
+      this.handleDragEnd();
```

Comment:
Any reason there is no event passed here?

Reply:
Oops! Thanks.